src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -53,4 +63,53 @@
         }
         return new ResponseEntity<>(userService.update(userDto, user), HttpStatus.ACCEPTED);
     }
+
+    @GetMapping("/child")
+    public ResponseEntity<List<ChildDto>> getChildren(Principal principal) {
+        return new ResponseEntity<>(userService.getChildren(getUser(principal.getName())), HttpStatus.OK);
+    }
+
+    @PostMapping("/child")
+    public ResponseEntity<HttpStatus> addChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}") List<@Valid ChildDto> childrenDtoList,
+                                                  Principal principal) {
+        if (childrenDtoList.stream().anyMatch(dto -> dto.getId() > 0)) {
+            throw new IllegalIdentifierException(getExceptionMessageSourceWithAdditionalInfo("exception.illegal.id", "Bigger than zero"));
+        }
+        userService.addChildren(getUser(principal.getName()), childrenDtoList);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @DeleteMapping("/child")
+    public ResponseEntity<HttpStatus> removeChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}") List<@NotNull Long> childrenIdToRemoveList,
+                                                     Principal principal) {
+        final User user = getUser(principal.getName());
+        final List<Long> userChildrenIdList = mapBy(user.getChildren(), Child::getId);
+        if (!userChildrenIdList.containsAll(childrenIdToRemoveList)) {
+            throw new IllegalIdentifierException(getExceptionMessageSource("exception.invalid.dto"));
+        }
+        userService.removeChildren(user, childrenIdToRemoveList);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @PutMapping("/child")
+    public ResponseEntity<HttpStatus> updateChildren(@RequestBody @Size(min = 1, message = "{exception.invalid.dto}") List<@Valid ChildDto> childrenDtoList,
```

Comment:
Would you mind to check that your IDE indicates 180 symbols as maximum and after to move all lines that bigger of such amount of symbols to the next line?